src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -1456,6 +1456,7 @@ - (id)accessibilityHitTest:(NSPoint)point withEnv:(JNIEnv *)env
 - (id)accessibilityFocusedUIElement
 {
     JNIEnv *env = [ThreadUtilities getJNIEnv];
+    GET_CACCESSIBILITY_CLASS_RETURN(nil);
```

Comment:
Is it caused by the JDK-8257853? Should we update the accessibilityIndexOfChild as well?

Reply:
It made it visible however it is just a good practice not to assume that class reference is already initialized before using it. So, yes, while i was not seeing any failures in accessibilityIndexOfChild  i might fix it as well.

Reply:
If this was missed in the JNF work, add the label "jnf" to the bug.

Reply:
Done.